gui/wxpython/location_wizard/wizard.py

```diff
@@ -72,6 +72,11 @@
 global wizerror
 global translist
 
+if globalvar.CheckWxVersion(version=[4, 1, 0]):
+    search_evt = wx.EVT_SEARCH
+else:
+    search_evt = wx.EVT_SEARCHCTRL_SEARCH_BTN
+
```

Comment:
This event is emitted when pressing the search button, right? Why not use (instead or in addition to) wx.EVT_TEXT to search as you type? That seems more intuitive.

Reply:
No button involved. Works on Mac with pressing enter with search field in focus. Just like it previously worked in Windows, where it actually worked. The question is whether it works with this fix in Linux and Windows.

Reply:
With second update commit wx.EVT_TEXT is used.